WSDebugger.pas

```diff
@@ -90,14 +96,19 @@ procedure OutputDebugString(Number: Integer; Elapsed: Int64);
     Exit;
 {$IF DEFINED(DEBUG) OR DEFINED(DEBUG_WS) OR DEFINED(CHECKSPEED)}
   LMsg := Format('%d: %d', [Number, Elapsed]);
-  {$IF DEFINED(MSWINDOWS) OR DEFINED(DEBUG_WS)}
-  Winapi.Windows.OutputDebugString(PChar(LMsg));
-  {$ENDIF}
-  {$IF DEFINED(ANDROID)}
-   if TPlatformServices.Current.SupportsPlatformService(IFMXLoggingService, LLogService) then
-     begin
-       LLogService.Log('%s', [LMsg]);
-     end;
+  {$IF DEFINED(DEBUG_WS)}
+    {$IF DEFINED(MSWINDOWS) }
+    Winapi.Windows.OutputDebugString(PChar(LMsg));
+    {$ENDIF}
+    {$IF DEFINED(ANDROID)}
+     if TPlatformServices.Current.SupportsPlatformService(IFMXLoggingService, LLogService) then
+       begin
+         LLogService.Log('%s', [LMsg]);
+       end;
+    {$ENDIF}
+    {$IF DEFINED(LINUX)}
+    Writeln(lMsg);
+    {$ENDIF}
```

Comment:
Same issue here.



Instead of
{$IF DEFINED(MSWINDOWS)}
{$ENDIF}
{$IF DEFINED(ANDROID)}
{$ENDIF}
{$IF DEFINED(LINUX)}
{$ENDIF}

use

{$IF DEFINED(MSWINDOWS)}
{$ELSEIF DEFINED(ANDROID)}
{$ELSEIF DEFINED(LINUX)}
{$ENDIF}